cmd/daemon/apis/shared.go

```diff
@@ -0,0 +1,30 @@
+package apis
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+
+	fluxevent "github.com/weaveworks/flux/event"
+)
+
+type Message struct {
+	TitleLink string
+	Body      string
+	Type      string
+	Title     string
+	Event     fluxevent.Event
+}
+
+// An exporter sends a formatted event to an upstream.
+type Exporter interface {
+	// Send a message through the exporter.
+	Send(c context.Context, client *http.Client, message Message) error
```

Comment:
What is the `http.Client` meant for? It's not used by any of the exporters implemented with this change?

Reply:
Leftover. I've removed it.